tests.txt

```diff
@@ -1 +1 @@
-@aws-cdk-testing/framework-integ/test/aws-s3/test/integ.bucket-auto-delete-objects.js
+@aws-cdk-testing/framework-integ/test/aws-s3/test/integ.bucket-auto-delete-objects.js
```

Comment:
curious why GH is highlighting this change but they seem to be identical?

Reply:
It was missing a newline. Sorry about that. It's fixed now